config/local-dev.yml

```diff
@@ -29,6 +29,12 @@ server:
   storage:
     endpoint: postgresql://rwuser:password@localhost:5432/indexer?sslmode=disable
     backend: postgres
+  node:
```

Comment:
Updated, but added the anchor one level down at `SourceConfig.nodes` since the analyzer `SourceConfig` has fields (eg cache) that aren't needed for the server source config.